cardboard/templates/tools.html

```diff
@@ -80,8 +80,8 @@ <h1>Tools and References</h1>
                 <div class="card-title"><h2>Image Puzzles</h2></div>
                 <div class="card-text">
                     <ul>
-                        <li><a href="https://chrome.google.com/webstore/detail/huntress-reverse-image-se/nkhjamjppefmeceaokadfacbdmmcmdpg?hl=en">Huntress Reverse Image Search Chrome Plugin</a></li>
-                        <li>Image Manipulation Programs: Photoshop, <a href="https://www.gimp.org/">GIMP</a>, MS Paint</li>
+                        <li><a href="https://chrome-stats.com/d/nkhjamjppefmeceaokadfacbdmmcmdpg/download">Huntress Reverse Image Search Chrome Plugin</a></li>

```

Comment:
fwiw I think Huntress is no longer being maintained / @kcaze himself recommends against using it. I use [this](https://chromewebstore.google.com/detail/search-by-image/cnojnbdhbhnkbcieeekonklommdnndci) these days

Reply:
would you mind swapping out the Huntress link for that newer one?

Reply:
Updated